### PR TITLE
Update features

### PR DESCRIPTION
**Pull request by andrjuha01**

### CODE REVIEW

File: lib/veeqo/request.rb [hunk 1]
```diff
@@ -63,6 +64,11 @@ def put(path, params = {})
         build_response_object response
       end
 
+      def quantity(path, params = {})
+        response = raw_request(:get, path, params)
+        response.headers['X-Total-Count'].to_i
```
Comment (yurkiv) [inline lib/veeqo/request.rb:69]: 👍

File: lib/veeqo/resources/order.rb [hunk 2]
```diff
@@ -54,5 +54,10 @@ def self.destroy(resource_id, params = {})
     rescue Veeqo::NotFound
```
Comment (yurkiv) [inline lib/veeqo/resources/order.rb]: As I understand, if body is empty, gem will not catch NotFound 
 error: https://github.com/coaxsoft/veeqo_api_ruby/pull/8/files#diff-d751a380d1e5ae4dc785cfc976ab72e7R53

**Review by yurkiv — APPROVED** [review]
Good code. Thanks for updating.

File: lib/veeqo/request.rb (hunk 1 above)
Comment (andrjuha01) [reply lib/veeqo/request.rb]: @yurkiv I think 0 as a total count makes more sense then just not found returned.
For example, I would like to know, how many requests are proceed to Veeqo to take all products with 100 items per page pagination.
I take Veeqo::Product.count / 100 and get that number.
If it produces 404 error, than I have to rescue that error. That's extra code here, but if it breaks gem's logic, I can raise error if X-Total-Count is 0.

File: lib/veeqo/resources/order.rb (hunk 2 above)
Comment (andrjuha01) [reply lib/veeqo/resources/order.rb]: Right now through api you can't destroy an order.
If body is empty, but status is present, then, by that status error is raised, if it's http error.

**Pull request merged**